sim: 1
D: 0.13
sleep_rate: 0.05
init_gain: 100

# 4
init_x: 0.311
init_y: 0.148
init_z: 0.052

# 3
# init_x: 0.2837
# init_y: 0.2641
# init_z: 0.2055

# 2
# init_x: 0.2637
# init_y: 0.2541
# init_z: 0.2155

# 1
# init_x: 0.3037
# init_y: 0.2641
# init_z: 0.2105